### PR TITLE
Traitor Uplink Content 

### PR DESCRIPTION
**Pull request by SweptWasTaken**
Adds about 30 items to the uplink. Also added a bunch of odds-and-ends to round out the actual content.

I also added some entries for stuff that I don't want to implement with this PR.

(The git commits are not a changelog, i've added a lot more than they show.)

### CODE REVIEW

File: Resources/Prototypes/Catalog/uplink_catalog.yml [hunk 1]
```diff
@@ -1,17 +1,238 @@
...
+# For the Mandella
+- type: uplinkListing
+  id: UplinkMagazineClRiflePistol
+  category: Ammo
+  itemId: MagazineClRiflePistol
+  price: 2
+
+# For the Inspector
+- type: uplinkListing
+  id: Uplink  SLMagnum
```
Comment (Tomeno) [inline Resources/Prototypes/Catalog/uplink_catalog.yml:98]: extra whitespace in id and itemId

File: Resources/Prototypes/Catalog/Fills/duffel.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 - type: entity
   parent: ClothingBackpackDuffelMedical
   id: ClothingBackpackDuffelSurgeryFilled
-  name: surgical duffel bag
+  name: surgical duffel duffelbag
```
Comment (Zumorica) [inline Resources/Prototypes/Catalog/Fills/duffel.yml:4]: duffel duffelbag?

File: Resources/Prototypes/Catalog/Fills/duffel.yml [hunk 3]
```diff
@@ -14,3 +14,129 @@
...
+        amount: 2
+#     - name: SMGSuppressor
+  - type: Storage
+    capacity: 100
+
+- type: entity
+  parent: ClothingBackpackDuffelSyndicate
+  id: ClothingBackpackDuffelSyndicateFilledLMG
+  name: L6 Saw bundle
+  description: "Swept's choice: The iconic L6 lightmachinegun, bundled with 2 box magazines."
```
Comment (Zumorica) [inline Resources/Prototypes/Catalog/Fills/duffel.yml:75]: Swept moment

**Comment by SweptWasTaken** [comment]
Addressed reviews, merge ASAP

**Review by DrSmugleaf — APPROVED** [review]

File: Resources/Prototypes/Entities/Objects/Specific/rehydrateable.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  components:
+  - type: SolutionContainer
+    contents:
+      reagents:
+      - ReagentId: chem.Nutriment
+        Quantity: 10
+    maxVol: 11 # needs room for water
+    caps: AddTo, RemoveFrom, FitsInDispenser
+  - type: Rehydratable
+    target: CarpMob_Content
```
Comment (20kdc) [inline Resources/Prototypes/Entities/Objects/Specific/rehydrateable.yml:34]: The prophecy has been fulfilled.

**Pull request merged**